http_parser.c

```diff
@@ -1297,26 +1392,62 @@ size_t http_parser_execute (http_parser *parser,
 
       case s_header_field:
       {
+        bool checkForTraditionalCase = false;
         const char* start = p;
         for (; p != data + len; p++) {
           ch = *p;
           c = TOKEN(ch);
 
-          if (!c)
+          if (!c) {
+            if (parser->header_state == h_complete_accept) {
+              parser->header_state = h_accept;
+            }
             break;
+          }
 
           switch (parser->header_state) {
             case h_general:
               break;
+            case h_A:
+              parser->index++;
+              switch (c) {
+                case 'c':
+                  parser->header_state = h_complete_accept;
+                  break;
+                case 'g':
+                  parser->header_state = h_matching_age;
+                  break;
+                case 'u':
+                  parser->header_state = h_matching_authorization;
+                  break;
+                default_h_general
```

Comment:
Ok, will change it back. But curious, you didn't like the name of macro or you feel macro is too much for this statement?